### ISSUE TITLE
HTTP client

### ISSUE THREAD

**Issue by cup** [gecko0307/dlib#160]
It other projects offer URL parsing and HTTP client:

- https://github.com/adamdruppe/arsd
- https://github.com/ikod/dlang-requests
- https://github.com/vibe-d/vibe.d

but it seems Dlib does not:

https://github.com/gecko0307/dlib/tree/master/dlib/network

is that the case or am I missing it?

**Comment by gecko0307** [comment]
Yes, dlib doesn't have built-in http yet, would be nice to have it.
